src/ajax.js

```diff
@@ -0,0 +1,150 @@
+/*
```

Comment:
I'll take it out before merging. I wasn't 100% certain with `corslite` yet, but if it goes in, then `ajax` goes out.